app/controllers/introduce_yourself_controller.rb

```diff
@@ -6,7 +6,7 @@ def update
 
     if @step.valid?
       app = current_or_new_snap_application
-      app.update!(snap_application_update_params)
+      app.save!
```

Comment:
yeah, before we were assigning the birthday to the `snap_application`, but now we want it on the `member` instead. This didn't become clear until I started filling out the PDF and realized we were being inconsistent about birthday. For regular members we were assigning birthday to the `member`, but for the primary member we were getting it from the `snap_application`